pegasus/sites.v3/code.org/public/css/interactive_map.css

```diff
@@ -0,0 +1,28 @@
+#map {
+  width: 700px;
+  min-height: 425px;
+  float: left;
+  margin-right: -100px;
+}
+
+#infobox {
+  text-align: left;
+  float: right;
+}
+
+.info {
+  font-size: 18px;
+}
+
+#state-dropdown {
+  width: 70%;
+  height: 34px;
+  background-color: rgb(226,228,227); color: rgb(89,89,89);
```

Comment:
@breville we've talked about whether we want colors to be RGB or Hex but I don't think we've ever made a decision. I want to move towards using hex everywhere and eventually replacing with [our color names](https://github.com/code-dot-org/code-dot-org/blob/staging/shared/css/color.scss) so it's documented that all of our colors are in this file and nothing else. Is that unreasonable? And what's the reason to use RGB > Hex?